### PR TITLE
bpo-32892: Update the documentation for handling constants in AST.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
I included these changes in the PR for other AST changes #9605, but it was not merged in 3.8.

https://bugs.python.org/issue32892

### CODE REVIEW

**Review by isidentical — APPROVED** [review]

File: Doc/library/ast.rst [hunk 1]
```diff
@@ -101,12 +101,16 @@ Node classes
...
+   Class :class:`ast.Constant` is now used for all constants.
+
 .. deprecated:: 3.8
 
-   Class :class:`ast.Constant` is now used for all constants. Old classes
-   :class:`ast.Num`, :class:`ast.Str`, :class:`ast.Bytes`,
+   Old classes :class:`ast.Num`, :class:`ast.Str`, :class:`ast.Bytes`,
    :class:`ast.NameConstant` and :class:`ast.Ellipsis` are still available,
-   but they will be removed in future Python releases.
+   but they will be removed in future Python releases.  In meanwhile,
```
Comment (isidentical) [inline Doc/library/ast.rst]:
```suggestion
   but they will be removed in future Python releases.  In the meanwhile,
```

File: Doc/library/ast.rst [hunk 2]
```diff
@@ -101,12 +101,16 @@ Node classes
...
+   instantiating them will return an instance of different class.
```
Comment (isidentical) [inline Doc/library/ast.rst]:
```suggestion
   instantiating them will return an instance of a different class.
```

**Review by isidentical — COMMENTED** [review]
These might clarify the doc a bit

**Comment by serhiy-storchaka** [comment]
Thank you for your corrections @isidentical! I always had problems with articles which are absent in my native language.

**Pull request merged**